src/vs/workbench/contrib/positronAssistant/common/interfaces/positronAssistantService.ts

```diff
@@ -93,10 +99,22 @@ export interface IPositronAssistantService {
 		onCancel: () => void,
 		onClose: () => void,
 	): void;
+
 	/**
 	 * Get the supported providers for Positron Assistant.
 	 */
 	getSupportedProviders(): string[];
+
+	/**
+	 * Add a language model configuration.
+	 */
+	addLanguageModelConfig(source: IPositronLanguageModelSource): void;
```

Comment:
More idiomatically I think this would be `registerLanguageModelConfig` and would return a `Disposable` for unregistering instead of having a separate `remove` method. Then you don't need to worry about object equality and can use disposable stores, etc. Up to you, though.